CIP-0072/README.md

```diff
@@ -0,0 +1,126 @@
+
+This hash is calculated by taking the entire metadata tree object, ordering the keys in the object alphanumerically, and then hashing the resulting JSON string using the blake2b hashing algorithm. The hash is encoded as a hex string.
+
+*`cip26`*: An array of URLs that point to the location of the dApp's metadata. The client should try to fetch the metadata from the first URL in the array. If the client fails to fetch the metadata from the first URL, it should try the second URL, and so on. If the client fails to fetch the metadata from all URLs, it should consider the metadata to be invalid.
+
+The format of the metadata tree object is described in the [CIP-26](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0026) specification.
+
+*`did`*(optional): The DID of the dApp's developer. The DID is used to verify the authenticity of the certificate. The client should use the DID to fetch the developer's public key from the Cardano blockchain. The client should then use the public key to verify the signature of the certificate. This field is *optional* and can be used by identity centric dApps to verify the authenticity of the developer.
+
+*`signature`*: The signature of the certificate. The signature is done over the blake2b hash of the certificate. The client should use the public key to verify the signature of the certificate. If a DID is provided, the client should use the DID to fetch the developer's public from the DID Document. If a DID is not provided, the client should use the public key provided in the signature object or from a already known public key.
+
+## Certificate JSON Schema
+```json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "subject": {
+      "type": "string"
+    },
+    "type": {
+      "type": "string",
+      "enum": [
+        "REGISTER",
+        "UPDATE"
+      ]
+    },
+    "rootHash": {
+      "type": "string"
+    },
+    "cip26": {
+      "type": "array",
+      "items": [
+        {
+          "type": "string"
+        }
+      ]
+    },
+    "did": {
+      "type": "string"
+    },
+    "signature": {
+      "type": "object",
+      "properties": {
+        "r": {
+          "type": "string"
+        },
+        "s": {
+          "type": "string"
+        },
+        "algo": {
+          "type": "string"
+        },
+        "pub": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "r",
+        "s",
+        "algo",
+        "pub"
+      ]
+    }
+  },
+  "required": [
+    "subject",
+    "rootHash",
+    "cip26",
+    "signature"
+  ]
+}
+```
+
+## Metadata Label
+
+When submitting the transaction metadata picks one of the following values for `transaction_metadatum_label`:
+
+- `666`: DApp Registration
+- `667`: DApp Certification
```

Comment:
❔ Why `666` and shall we chose a less conotated number?

Reply:
Fair :+1:  ; Picked a less conotated number that was available according to the https://github.com/cardano-foundation/CIPs/blob/master/CIP-0010/registry.json